spec/conformance/capability-sets/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "SMART App Launch: Capability Sets"
+layout: default
+---
+## Capability Sets
+
+The following four Capability Sets are defined. Any individual SMART server will publish a granular list of its capabilities; from this list you can determine which of these Capability Sets are supported:
```

Comment:
May want to avoid the 2nd person -- "a client can determine" rather than "you can determine".

Reply:
.md formatting nitpick: @Healthedata1 could you configure your editor to use 80-character lines and wrap across lines? It looks here like paragraphs are being slurped into single lines.

Reply:
fixed, updated editor